Swift_Alert/Swift_Alert/Helpers/BottomSheet.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  BottomSheet.swift
+//  Swift_Alert
+//
+//  Created by coder on 02.02.2023.
+//
+
+import SwiftUI
+
+/// Универсальный боттомшит
+struct BottomSheetView<Content: View>: View {
+
+    // MARK: - Public Propeties
+    
+    let contentClosure: () -> Content
```

Comment:
```suggestion
    let contentHandler: () -> Content
```
у тебя ниже есть actionHandler значит надо к 1 виду привести все